src/odb/include/odb/db.h

```diff
@@ -1694,6 +1694,11 @@ class dbBlock : public dbObject
 
   void clearUserInstFlags();
 
+  ///
+  /// return the module name id map
+  ///
+  std::map<std::string, int>* getModuleNameIdMap();
```

Comment:
This is an internal implementation detail and doesn't need to be in the public interface.